lib/standard/collection/abstract_collection.nit

```diff
@@ -66,6 +66,14 @@ interface Collection[E]
 	#     assert [1..1[.is_empty   == true
 	fun is_empty: Bool do return length == 0
 
+	# Alias for `not is_empty`.
```

Comment:
Do we need to duplicate all boolean services from now?

I'm not a big fan of that one...